.github/workflows/ci.yml

```
@@ -30,12 +34,195 @@ jobs:
+        if: matrix.os == 'windows-latest'
+        run: |
+          # Create obj directory to match expected artifact path
+          New-Item -ItemType Directory -Force -Path "obj"
+
+          # Find MSBuild
+          $msbuild = "${env:ProgramFiles}\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\MSBuild.exe"
+          if (-not (Test-Path $msbuild)) {
+            Write-Error "MSBuild not found at $msbuild"
+            exit 1
+          }
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)